tasks/run_puppet_device.json

```diff
@@ -10,6 +10,10 @@
     "timeout": {
       "description": "The timeout in seconds for the puppet device command(s), defaults to 64",
       "type": "Optional[Variant[Integer, Pattern[/^[0-9]+$/]]]"
+    },
+    "apply": {
+      "description": "The path to the Puppet manifest to apply which is located on the (proxy) Puppet agent",
+      "type": "Optional[Pattern[/.+\.pp$/]]"
     }
```

Comment:
Maybe:

```
      "description": "The path to the Puppet manifest (on the (proxy) Puppet agent) to apply",
```

I suggest adding something to the README about this (the manifest must exist on the proxy) requirement.